### ISSUE TITLE
Cop idea: Redundant `Float::INFINITY` for ranges

### ISSUE THREAD

**Issue by ydakuka** [rubocop/rubocop#12545]
## Describe the solution you'd like

```ruby
# bad
(10..Float::INFINITY)

# good
(10..)
```

**Comment by koic** [comment]
They are not compatible:

```ruby
(10..Float::INFINITY).end # => Infinity
(10..).end                # => nil
```
